test_data/yast/nvme.yaml

```diff
@@ -0,0 +1,43 @@
+---
+nvm_disk: nvme0n1
```

Comment:
in case you forget to also rename to 'nvme' each key I would mention it here again, but if it is your choice is ok.

Reply:
you mean nvm_disk-> nvme_disk?

Reply:
yes, and for the rest of them below, alternative would look nice to group them under one nvme key, which would shorten the names of some of them.

Reply:
not a bad idea. but i would like to avoid it for now. it seems minor. but i will keep it in mind.